public/locales/en/translation.json

```
@@ -239,6 +240,9 @@
     "customTime": "Custom time",
     "cancelTimer": "Cancel timer"
   },
+  "BoardReactionsMenu": {
+    "react": "React {{reaction}}"
```

```suggestion
    "react": "React with {{reaction}}"
```
(not sure about this)